builder/rootfs/etc/confd/templates/check-repos

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
+set -x
```

Comment:
`set -x` shouldn't be left on in production; there's already a lot of spew in the builder logs.